app/System/Report/Services/ReportService.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace App\System\Report\Services;
+
+use App\System\Report\DTO\ReportDTO;
+use App\System\Report\Repositories\ReportRepositoryInterface;
+
+class ReportService
+{
+    public function __construct(private ReportRepositoryInterface $reportRepository)
+    {
+    }
+
+    public function getReport(string $airport, string $contentType)
+    {
+        $format = $this->getOutputFormat($contentType);
+
+        $reportDTO = new ReportDTO($airport, $format);
+
+        return $this->reportRepository->getReport($reportDTO);
+    }
+
+
+    private function getOutputFormat(string $format): string
+    {
+        $allowedFormats = config('reports.allowedFormats');
```

Comment:
In big projects you shouldn't use helpers like config, they are infrastructure level dependencies and they break your  Unit testing.

Inject configs from the container